oceannavigator/frontend/yarn.lock

```diff
@@ -1979,21 +1959,11 @@ css-select@^4.1.3:
     domutils "^2.6.0"
     nth-check "^2.0.0"
 
-css-what@^5.0.0, css-what@^5.0.1:
+css-what@^5.0.0:
```

Comment:
you can see here that `css-what` is already marked as a dependency by another package we use...no need to explicitly add it in `package.json`.